src/configure/configure.ts

```diff
@@ -511,13 +526,8 @@ class Orchestrator {
             //Get languageSettings (corresponding to language of selected settings) provided by RepoAnalysis
             await this.updateRepositoryAnalysisApplicationSettings(repoAnalysisResult);
         }
+        telemetryHelper.setTelemetry(TelemetryKeys.ChosenTemplate, this.inputs.pipelineConfiguration.template.label);
```

Comment:
this also

Reply:
The label gets decided and that's why we can store the telemetry as well.

Reply:
The label is a morphed label. it maps to more than one template. The template is decided only after web app is chosen. and that's where the telemetry should be set.

Reply:
Updated